.github/workflows/pull-request-management.yml

```diff
@@ -471,14 +464,16 @@ jobs:
     steps:
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.6 # Ancient version to match galaxy.ansible.com.
+          python-version: |
+            3.10
       - uses: actions/checkout@v3
       - name: Install requirements
         # Install the specific version of galaxy-importer used on galaxy.ansible.com
-        # The old version conflicts with our versions of other tooling,
+        # The version conflicts with our versions of other tooling,
         # so we let the galaxy-importer version resolve remaining requirements.
+        # Also install deepmerge and jsonschema for use in rebuilding schemas.
         run: |
-          pip install "galaxy-importer==0.4.0.post1"
+          pip install "galaxy-importer==0.4.19" "jsonschema>=4.5.1,<4.18" "deepmerge>=1.1.0" "pydantic"
```

Comment:
Can we make this a file instead? we are tracking versions dependencies all over the place.
Also it says pydantic line 476 but not in the comment line 474 - is it on purpose?

Reply:
but it means we need to maintain this file as well with dependencies versions

Reply:
yes for this special case where we have to match whatever Ansible Galaxy decides is the current version.

Reply:
refactored